ios/RNSConvert.h

```diff
@@ -1,6 +1,7 @@
 #ifndef RNSConvert_h
```

Comment:
Are those 2 first lines needed?

Reply:
Indeed, they are not. Removed them already.